lib/utils/getCustomProps.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const csstree = require('css-tree');
+
+module.exports = (css) => {
+  // Find all root (html, body, :root) nodes
+  const rootNodes = csstree
+    .findAll(
+      csstree.parse(css),
+      (node) =>
+        node.type === 'Rule' &&
+        node.prelude.children.some((sel) =>
+          sel.children.some((child) =>
+            ['html', 'root', 'body'].includes(child.name),
+          ),
+        ),
+    )
```

Comment:
Is this too specific? It will literally only find custom props declared in `html`, `:root`, or `body` rules (or some combination, e.g. `html, :root {...}`).

Reply:
I think it's a reasonable limitation for an automated approach, and we could also allow projects to pass in a custom design tokens file that they generate themselves if their general CSS doesn't follow this pattern.

Reply:
Added in https://github.com/oddbird/sassdoc-theme-herman/commit/d5f6a2b4104c078414f2603781d0ce38c7467193